importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -4925,7 +4926,7 @@ Note* MusicXMLParserPass2::note(const QString& partId,
             else if (_e.name() == "stem")
                   stem(stemDir, noStem);
             else if (_e.name() == "type") {
-                  small = _e.attributes().value("size") == "cue" | _e.attributes().value("size") == "grace-cue";
+                  small = _e.attributes().value("size") == "cue" || _e.attributes().value("size") == "grace-cue";
```

Comment:
Better also add parantheses to the expression?

Reply:
It's not necessary (equality operators have precedence over logical OR, and bitwise OR for that matter. But if it'll help everyone sleep at night I'm happy to!

Reply:
I added them first to stop the compiler warning, then detected the | vs || bug and fixed that too.

Reply:
I checked now thogh: it does. Still those parens won't harm. OTH it shows that compiler warnings sometimes even point at a bug ;-)